services/app-web/src/common-code/healthPlanFormDataType/healthPlanFormData.ts

```diff
@@ -64,7 +64,18 @@ const hasValidRates = (sub: LockedHealthPlanFormDataType): boolean => {
                       rateInfo.rateDateStart !== undefined &&
                       rateInfo.rateDateEnd !== undefined &&
                       (rateInfo.rateProgramIDs !== undefined ||
-                          rateInfo.rateProgramIDs !== [])
+                          rateInfo.rateProgramIDs !== []) &&
+                      rateInfo.actuaryContacts.every(
+                          (actuaryContact) =>
+                              actuaryContact.name !== undefined &&
+                              actuaryContact.titleRole !== undefined &&
+                              actuaryContact.email !== undefined &&
+                              ((actuaryContact.actuarialFirm !== undefined &&
+                                  actuaryContact.actuarialFirm !== 'OTHER') ||
+                                  (actuaryContact.actuarialFirm === 'OTHER' &&
+                                      actuaryContact.actuarialFirmOther !==
```

Comment:
This is kind of confusing, let me know if anyone has a suggestion. Every contact should have every field, if `actuarialFirm` is `OTHER` then `actuarialFirmOther` should not be undefined.

Reply:
Outside the scope of this PR, we need to rework this function.

l am very wary of expanding the way we are using this hasValidateRates function to include checking nested fields. We aren't unit testing these functions either and these functions block/allow submission.

The chance for bugs increases as we load logic in here. And it might not be needed - we already have multiple validation schemas in the app. Kind of feels like we should be reading from one of those schema to check  rates - not hard coding the validation data in yet another place.  

Since this is new functionality (we never had isValidContacts) and never checked actuary contacts at this stage before, I suggest we do something very simple for now and revisit with a follow on PR. Just spot check that the actuary contacts have email fields and an actuary firm exists, that's it. 

 @rswerve do you have thoughts here? 

@rswerve